ca_tools/scripts/control_node.py

```diff
@@ -44,142 +39,133 @@ class ctrl_Node:
     def __init__(self):
         
         rospy.init_node('control_node')
-        pubtopic = "/create1/cmd_vel"
-        self.my_pub = rospy.Publisher(pubtopic, Twist, queue_size=10)
-        subtopic = "/create1/gts"
+    
+        self._my_pub = rospy.Publisher(pubtopic, Twist, queue_size=10)
+
         # Create a subscriber with appropriate topic, custom message and name of callback function.
-        self.my_sub=rospy.Subscriber(subtopic,Odometry,self._callback)
-        self.my_goals = []
+        self._my_sub=rospy.Subscriber(subtopic,Odometry,self._callback)
+        self._my_goals = []
         
         for i in range(4):
-            self.my_goals.append(Pose2D())
+            self._my_goals.append(Pose2D())
 
 
-        self.my_goals[0].x=0.0
-        self.my_goals[0].y=0.0
+        self._my_goals[0].x=0.0
```

Comment:
As I mentioned before, there are some sets here that are you needed.

Reply:
Adressed